website changes

ptothej23 opened the pull request:
- change order of leadership to alphabetical
- removed map header from teams page
- updated many team members' statuses to "past"
- changed title, links to, and path of "Publications" page to "Research"
  - updated references to this, including moving directory `content/about/publications/` to `content/about/research/`
- shortened instances of "Delphi* Milestones" to just "Milestones"

melange396 commented on content/people/index.md:
```diff
@@ -345,7 +345,8 @@ people:
   - highlight
-  leaderOrder: 3
+  leaderOrder: 1
+note: Executive Director
```
```suggestion
  note: Executive Director
```

carlynvandyke commented:
@ptothej23 At some point (long ago) we discussed cleaning up/re-orging this page and the directive I got was that we should check with everyone before we moved them from "core"/"contributor" to "past," to avoid any potential offense (and sending all those emails seemed like a lot so it got buried at the bottom of my to-do list). What are your thoughts about that now?

ptothej23 commented:
Fair point.

Give me till early next week to validate all of this with Ryan (these were
mostly his suggestions) and I can take the task of communicating this to
folks.

Peter


On Mon, Mar 24, 2025, 5:45 PM Carlyn Van Dyke ***@***.***>
wrote:

> @ptothej23 <https://github.com/ptothej23> At some point (long ago) we
> discussed cleaning up/re-orging this page and the directive I got was that
> we should check with everyone before we moved them from
> "core"/"contributor" to "past," to avoid any potential offense (and sending
> all those emails seemed like a lot so it got buried at the bottom of my
> to-do list). What are your thoughts about that now?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cmu-delphi/www-main/pull/1074#issuecomment-2749473127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAGM4WCNSFPECOZLI54W3AL2WB4GZAVCNFSM6AAAAABZVGVZWWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBZGQ3TGMJSG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
> [image: carlynvandyke]*carlynvandyke* left a comment
> (cmu-delphi/www-main#1074)
> <https://github.com/cmu-delphi/www-main/pull/1074#issuecomment-2749473127>
>
> @ptothej23 <https://github.com/ptothej23> At some point (long ago) we
> discussed cleaning up/re-orging this page and the directive I got was that
> we should check with everyone before we moved them from
> "core"/"contributor" to "past," to avoid any potential offense (and sending
> all those emails seemed like a lot so it got buried at the bottom of my
> to-do list). What are your thoughts about that now?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cmu-delphi/www-main/pull/1074#issuecomment-2749473127>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BAGM4WCNSFPECOZLI54W3AL2WB4GZAVCNFSM6AAAAABZVGVZWWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONBZGQ3TGMJSG4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
